library/src/webapp/js/lang-datepicker/lang-datepicker.js

```diff
@@ -1850,9 +1876,36 @@ $.datepicker._gotoToday = function (id) {
 			// set the initial date for the picker
 			initDateTime();
 
-			// instantiate datetimepicker
+			// Instantiate datetimepicker
 			localDTPicker = $(options.input).css('min-width','200px').datetimepicker(cfg);
 
+			var openDatepickerLabel = $.datepicker._defaults['openDatepicker'] || 'Open Datepicker';
```

Comment:
Can you not use const here?

Reply:
I appreciate your suggestion to use const for variable declarations. The usage of `var` in this instance was to maintain consistency with the existing code in this file. and I like var. Do i have to update it to `const`?

Reply:
I would always use const with a new line of code. It makes it clear it can't be hoisted etc. There are reasons they introduced it.